projects/ion/src/lib/tab-group/tab-group.component.ts

```diff
@@ -11,7 +19,7 @@ import {
   templateUrl: './tab-group.component.html',
   styleUrls: ['./tab-group.component.scss'],
 })
-export class IonTabGroupComponent implements OnInit {
+export class IonTabGroupComponent implements OnInit, OnChanges {
   @Input() tabs: TabInGroup[];
   @Input() direction: DirectionType = 'horizontal';
   @Input() border: BorderDirectionType;
```

Comment:
Maybe you can already set a default border value here.

```suggestion
  @Input() border: BorderDirectionType = 'bottom';
```